### DISCUSSION TITLE
Difference between nerfstudio's instant-ngp and the original implementation

### DISCUSSION THREAD

**Discussion by balintfodor** [nerfstudio-project/nerfstudio#2395] [Q&A]
When I played on my dataset I noticed that the original [instant-ngp code](https://github.com/NVlabs/instant-ngp) runs faster and produces better results (less foggy, maybe a bit a sharper) with the default parameters than nerfstudio's instant-ngp implementation which I believe has similar default parameters (?), except the number of steps 30k vs 35k.

I tried to reduce the extra evaluation computation to get a better performance for the nerfstudio's implementation. I also tried to spot any subtle differences in the parameters without any luck. Used `images_on_gpu` too. I'm with a T4 (16GB) GPU, as I understand this is not the most ideal card for these purposes (in the original instant-ngp paper they use an RTX 3090) but that's what I have right now.

Can you point me to directions about this performance difference? Either parameters, code lines, or some explanation would be super helpful.

**Comment by HillYoungs** [comment]
Also want to know
